exercises/matching-brackets/introduction.md

```diff
@@ -0,0 +1,8 @@
+# Introduction
+
+You're given the opportunity to write software for the Bracketeer™, an ancient but powerful mainframe.
```

Comment:
I'm not a lawyer and all that but is slapping a (TM) on a hypothetical story example in any way a problem? AFAIK there's no actual trademark here but this "suggests" there might be. Or not?

Reply:
Unregistered trademarks [don't have to use (TM)](https://en.wikipedia.org/wiki/Unregistered_trademark), and IIRC, it is on the trademark claimant to pursue enforcement via suing or other means, and that ability is sharply limited.

Here's a paragraph describing the enforcement in the US:

> _In the [United States](https://en.wikipedia.org/wiki/United_States), neither federal nor state registration is required to obtain common law trademark protection, albeit the protection may be limited. In contrast to federal registration, common law trademarks are usually enforceable only within the geographic region or locale where the trademark owner is using it in business. When an infringement occurs, an unregistered trademark owner may not be able to sue and collect damages or recover attorneys fees. In those jurisdictions with limited protection to unregistered trademark owners, a common law trademark owner's remedies may be limited to injunctive relief (a court order for the defendant to cease and desist the infringement)._